vital/algo/algorithm_factory.h

```diff
@@ -145,6 +146,38 @@ bool has_algorithm_impl_name(std::string const& type_name,
 #define ADD_ALGORITHM( impl_name, conc_T)                    \
   add_factory( new kwiver::vital::algorithm_factory_0<conc_T>( conc_T::static_type_name(), impl_name ))
 
+// ============================================================================
+class algorithm_registrar
```

Comment:
Some basic doc strings for this class would be useful.

Reply:
done